airbyte-scheduler/src/main/java/io/airbyte/scheduler/SchedulerApp.java

```diff
@@ -95,9 +115,19 @@ public void start() throws IOException {
     final ScheduledExecutorService scheduledPool = Executors.newSingleThreadScheduledExecutor();
     final WorkerRunFactory workerRunFactory = new WorkerRunFactory(workspaceRoot, pbf);
 
+    final WorkflowOptions workflowOptions = WorkflowOptions.newBuilder()
```

Comment:
should we hide it behind some sort of temporal Airbyte client? So the code isn't sprinkled with all these new classes & config?